script/openqa-dump-templates

```diff
@@ -179,24 +185,41 @@ sub product_key ($r) {
 }
 
 if ($tables{'JobGroups'}) {
-    my $group = (keys %{$options{group}})[0];
-    $url->path($options{apibase} . '/job_templates_scheduling/' . ($group // ''));
-    my $res = $client->get($url)->res;
-    handle_error($group // 'all groups', $url, $res) unless $res->code && $res->code == 200;
-    if ($group) {
-        # This is already the YAML document of a single group
-        push @{$result{JobGroups}}, {group_name => $group, template => $res->json};
+    my $group = defined $options{group} ? (keys %{$options{group}})[0] // '' : '';
+    if ($options{convert}) {
+        # use job_templates_scheduling, which converts if no YAML
+        # string already exists
+        $url->path($options{apibase} . '/job_templates_scheduling/' . $group);
+        my $res = $client->get($url)->res;
```

Comment:
Maybe keep this line out of the if/else and delete it from  https://github.com/os-autoinst/openQA/pull/6446/files#diff-5a11940e46ad506a2a695c580813fc76ce08bcaa2f2a8ee26db6082b877be8daR209